modules/37_industry/subsectors/datainput.gms

```diff
@@ -436,15 +471,28 @@ pm_tau_ces_tax(t,regi,"feh2_steel")     =  50 * sm_TWa_2_MWh * 1e-12;
 $endif.cm_subsec_model_steel
 pm_tau_ces_tax(t,regi,"feh2_cement")    = 100 * sm_TWa_2_MWh * 1e-12;
 
+*` temporal phase-in of mark-up cost changes defined by cm_CESMkup_ind
+*` no changes to mark-up cost before 2025, then gradual phase-in until 2040
+p37_CESMkup_policy_phasein(t) = 0;
+p37_CESMkup_policy_phasein(t)$(t.val ge 2040) = 1;
+p37_CESMkup_policy_phasein(t)$(t.val eq 2035) = 1/2;
+p37_CESMkup_policy_phasein(t)$(t.val eq 2030) = 1/4;
+p37_CESMkup_policy_phasein(t)$(t.val eq 2025) = 1/8;
```

Comment:
This change should be documented in `main.gms` with the definition of `cm_CESMkup_ind`.